src/test/java/calculator/FixedDelimiterCalculator.kt

```diff
@@ -0,0 +1,28 @@
+package calculator
+
+import java.lang.IllegalArgumentException
+
+private val FIXED_DELIMITERS = listOf(":", ",")
+
+class FixedDelimiterCalculator(
+    private val calculatorNumberRangeValidator: CalculatorNumberRangeValidator
+) : StringCalculateStrategy {
+    override fun support(text: String?): Boolean =
+        text != null && hasFixedDelimiter(text)
+
+
+    override fun calculate(text: String?): Int {
+        if (text == null) {
+            throw IllegalArgumentException("empty text")
+        }
```

Comment:
require()는 값을 만족하지 않으면 IllegalArgumentException을 발생시킵니다.

```suggestion
        require(text != null) { "empty text" }
```